dialogue-target/src/main/java/com/palantir/dialogue/Response.java

```diff
@@ -22,19 +22,23 @@
 import java.util.List;
 import java.util.Optional;
 
-public interface Response extends Closeable {
+// TODO(1234): probs not want abstract
+public abstract class Response implements Closeable {
```

Comment:
This can't merge, it would be an ABI break that causes consumers to fail at runtime if they compiled against the interface.

Reply:
Absolutely, just was quickest way to compile. Default method won't really work, cause there isn't really a good implementation here, since interfaces can't have instance variables.